python/lsst/ip/isr/ptcDataset.py

```diff
@@ -471,6 +473,11 @@ def fromTable(cls, tableList):
             inDict['finalMeans'][ampName] = record['FINAL_MEANS']
             inDict['badAmps'] = record['BAD_AMPS']
             inDict['photoCharge'][ampName] = record['PHOTO_CHARGE']
+            if calibVersion == 1.0:
```

Comment:
Can you add a docstring to the class that mentions that v1.1 adds the `ptcTurnoff` attribute?  I think it'll be easier to understand changes if it doesn't require looking for code tests on `calibVersion.`

Reply:
Agree. On a more general note, are the docstrings the only places where version changes are tracked? I just want to make sure that we don't have to update another document (or perhaps this is the time to decide if we want o create + maintain such an external document).